### DISCUSSION TITLE
Sugestão para implementar o método aparar()

### DISCUSSION THREAD

**Discussion by JohnPetros** [DesignLiquido/delegua#371] [Ideias]
Em JS há o método `trim`, que retorna uma nova string, mas sem os espaços em branco da string original, tanto do seu início, quanto do seu fim:

```
const str = " Olá, mundo! ";

const newStr = str.trim();
console.log(newStr); // "Olá, mundo!"
```
<br>
Não sei se já existe implementado em Delégua, já que não encontrei na wiki, mas proponho a criação do método `aparar`, que tenha o mesmo comportamento do `trim`.

Mas vocês podem sugerir outros nomes também.

**Comment by samuelrvg** [comment]
@JohnPetros Gostei da sugestão e do nome, o que você acha @leonelsanchesdasilva ?

Já até fiz aqui um exemplo, confirmando esse formato, eu já abro a PR.

**Reply by samuelrvg** [reply]
PR aberta já. 

https://github.com/DesignLiquido/delegua/pull/372

**Comment by samuelrvg** [comment]
Publicado versão com essa implementação.

[Versão 0.20.7](https://github.com/DesignLiquido/delegua/releases/tag/0.20.7)
